scipy/stats/_continuous_distns.py

```diff
@@ -1287,15 +1291,14 @@ class exponnorm_gen(rv_continuous):
 
     .. math::
 
-        f(x, K) = \frac{1}{2K} \exp\left(\frac{1}{2 K^2}\right) \exp(-x / K)
+        f(x, K) = \frac{1}{2K} \exp\left(\frac{1}{2 K^2} - x / K \right)
                   \text{erfc}\left(-\frac{x - 1/K}{\sqrt{2}}\right)
 
-    where the shape parameter :math:`K > 0`.
+    where :math:`x` is a real number and :math:`K > 0`.
 
-    It can be thought of as the sum of a normally distributed random
-    value with mean ``loc`` and sigma ``scale`` and an exponentially
-    distributed random number with a pdf proportional to ``exp(-lambda * x)``
-    where ``lambda = (K * scale)**(-1)``.
+    It can be thought of as the sum of a standard normal random
+    variable and an exponentially distributed random variable with rate
+    ``1/K``.
```

Comment:
In wikipedia and previous definition loc and scale are only used for the normal part.
How does this match up with the loc and scale shift that we have generically.
i.e. there should be a some explanation about the effect of scale, my guess similar to the old definition of lambda
or how do we need to choose k if we have lambda and normal-scale?

Reply:
In standard form, with X standard normal and Y exponential with rate 1/K , `X + Y` is `exponnorm`, thus `loc + scale*(X + Y)` is the sum of `norm(loc, scale)` and `scale * Y` and the rate parameter of the exponential distribution scales as stated in the old documentation.

I think it is better to state the standard form only this point, the scaling is explained after the notes with a reference to wikipedia.